coalahtml/_coalahtml/app/controllers/logs.js

```diff
@@ -9,5 +9,5 @@ angular.module('coalaHtmlApp')
         $scope.data = response.logs;
       });
     };
-    parseCoalaProject("tests/test_projects/simple");
+    parseCoalaProject("data");
```

Comment:
@Uran198 So the plan is like this:
Let's say I have a project `XYZ` with 2 cpp files:
```
XYZ
    .coafile
    file1.cpp
    file2.cpp
```
Now when the user runs coala-html it will create:
```
XYZ
    .coafile
    file1.cpp
    file2.cpp
    coalahtmlapp
        app
           views
           controllers
           styles
        data
        index.html
```
So, the reason this `data` is hardcoded is because this file will be moved appropriately to the parent of the data folder that will be created by coala-html.

Does that make sense ?